--- robojournal.orig	2013-09-03 02:16:01.662950581 +0000
+++ robojournal.fixed	2013-09-03 02:16:52.875642699 +0000
@@ -156,13 +156,7 @@
     icon.path = /usr/share/icons
     icon.files = robojournal64.png
 
-    shortcut-deb.path = /usr/share/menu
-    shortcut-deb.files = menus/robojournal
-
-    icon-deb.path = /usr/share/pixmaps
-    icon-deb.files = menus/robojournal.xpm
-
-    INSTALLS += target shortcut icon shortcut-deb icon-deb doc man
+    INSTALLS += target shortcut icon doc man
 
 }
 
